real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/GpsNode.java

```diff
@@ -86,6 +86,7 @@ private double convertMinSecToFloat_Longitude(String dddmmmmm) {
 		return ddd + mins / 60.0;
 	}
 	
+	/**{@inheritDoc}*/
 	@Override
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method peel() has an NPath complexity of 4608](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1781616641/issues/source?bid=2938665&fileBranchId=3059924#l91)